trio/_core/_run.py

```diff
@@ -1013,6 +1021,18 @@ def parent_nursery(self):
         """
         return self._parent_nursery
 
+    @property
+    def eventual_parent_nursery(self):
+        """The nursery this task will be inside after it calls
+        ``task_status.started()``.
+
+        If this task has already called ``started()``, or if it was not
+        spawned using `nursery.start() <trio.Nursery.start>`, then
+        `eventual_parent_nursery` is the same as `parent_nursery`.
```

Comment:
I think I have a mild preference for "if the task isn't in the middle of `start` shenanigans, then `eventual_parent_nursery` is `None`". What do you think?

Reply:
I'm indifferent between the two so will change it to suit your preference!